File: 3300_Minimum_Element_After_Replacement_With_Digit_Sum/solution.py
class Solution:
    def minElement(self, nums: List[int]) -> int:
        for i in range(len(nums)):
            s = 0
            while nums[i] > 0:
                s += nums[i] % 10
                nums[i] //= 10
            nums[i] = s
        return min(nums)

File: 3300_Minimum_Element_After_Replacement_With_Digit_Sum/solution.c
int minElement(int* nums, int n) {
    int m = INT_MAX;
    for (int i = 0; i < n; i++) {
        int s = 0;
        while (nums[i] > 0) { s += nums[i] % 10; nums[i] /= 10; }
        nums[i] = s;
        m = nums[i] < m ? nums[i] : m;
    }
    return m;
}
